initfiles/etc/DIR_NAME/environment.xml.in

```diff
@@ -902,6 +902,7 @@
                     parallelRequestLimit="20"
                     throttleCPULimit="75"
                     throttleDelayMs="5000"
+                    allowWriteFlush="true"
```

Comment:
If this setting becomes an expert setting, ops. can configure it via Software/Globals and it should be generated automatically as an expert setting, such that it can be picked by : getComponentConfigSP()->getPropInt("expert/@defaultJFileExtraFlags")